lib/controllers/services/invitation/functions/_get_impl.dart

```diff
@@ -65,17 +65,17 @@ Future<List<InvitationModel>> _fetchLocal({
 
 Future<List<InvitationModel>> _fetchFromBackend({
   String? senderID,
-  String? invitationID,
   String? recipientID,
+  String? invitationID,
   bool forceGet = false,
 }) async {
   List<InvitationModel> invitations = [];
   DbCollection collection =
       Database.instance.collection(InvitationController._collectionName);
 
   SelectorBuilder selectorBuilder = SelectorBuilder();
-  if (senderID != null) {
-    selectorBuilder.eq(InvitationFields.senderID.name, senderID);
+  if (recipientID != null) {
```

Comment:
receipentID is twice
SenderID is missing... try not to change the backend methods

Reply:
Sorry,there was a problem in one merge conflict....every method was having two recipientID i changed for other method....didnt notice this one